fastlane/lib/fastlane/actions/get_managed_play_store_publishing_rights.rb

```diff
@@ -26,7 +26,7 @@ def self.run(params)
         # Login
         credentials = JSON.parse(json_key_data)
         callback_uri = 'https://fastlane.github.io/managed_google_play-callback/callback.html'
-        uri = "https://play.google.com/apps/publish/delegatePrivateApp?service_account=#{credentials['client_email']}&continueUrl=#{URI.escape(callback_uri)}"
+        uri = "https://play.google.com/apps/publish/delegatePrivateApp?service_account=#{credentials['client_email']}&continueUrl=#{URI.encode_www_form_component(callback_uri)}"
```

Comment:
I believe this is now correct and it wasn't - query param should be encoded as special characters like `://` shouldn't be part of the param, right?